Library/Formula/premake.rb

```diff
@@ -1,8 +1,8 @@
 class Premake < Formula
-  desc "Premake is a build script generator"
-  homepage "http://industriousone.com/premake"
-  url "https://downloads.sourceforge.net/project/premake/Premake/4.3/premake-4.3-src.zip"
-  sha256 "36536490f8928d8ecde135da80cd8b751ea5bebe50cabba5c0de49cd41cb2780"
+  desc "Write once, build anywhere Lua-based build system"
+  homepage "https://premake.github.io/"
+  url "https://downloads.sourceforge.net/project/premake/Premake/4.4/premake-4.4-beta5-src.zip"
```

Comment:
It might be worth checking on that upstream and asking why it's been labelled a beta.